.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,24 @@
+name: Github-Actions
+
+on:
+  push:
+    branches: [ "github-actions" ]
+  pull_request:
+    branches: [ "github-actions" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.14.x
+
+    - name: Run Script
+      run:  make all
```

Comment:
Do we assume `make` is available or need to install any dependency?

Reply:
Here I assume make is available.